qrenderdoc/Windows/EventBrowser.cpp

```diff
@@ -103,25 +103,25 @@ void EventBrowser::OnLogfileLoaded()
   framestart->setData(COL_CURRENT, Qt::UserRole, QVariant(false));
   framestart->setData(COL_FIND, Qt::UserRole, QVariant(false));
   framestart->setData(COL_BOOKMARK, Qt::UserRole, QVariant(false));
-  framestart->setData(COL_SELECT_EID, Qt::UserRole, QVariant(0));
+  framestart->setData(COL_LAST_EID, Qt::UserRole, QVariant(0));
 
   uint lastEID = AddDrawcalls(frame, m_Ctx->CurDrawcalls());
-  frame->setData(COL_EID, Qt::UserRole, QVariant(lastEID));
-  frame->setData(COL_SELECT_EID, Qt::UserRole, QVariant(lastEID));
+  frame->setData(COL_EID, Qt::UserRole, QVariant(0));
+  frame->setData(COL_LAST_EID, Qt::UserRole, QVariant(lastEID));
 
   ui->events->insertTopLevelItem(0, frame);
 
   ui->events->expandItem(frame);
 
-  m_Ctx->SetEventID(this, lastEID);
+  m_Ctx->SetEventID(this, 0, lastEID);
```

Comment:
This is a bit tricky. I'm not sure we can pass in 0, as 0 isn't a valid eventID so things that rely on having a valid eventID might break down (possibly nothing does at the moment).

On the other hand we have the frame node selected which is kind of outwith the list of events. Perhaps we should pass lastEID for both here, since the last even should be a `Present()` type marker? Does that cause any problems with the event browser behaviour?